### PR TITLE
usermanual_en.html

### PR DESCRIPTION
**Pull request by octaeder**
- Added section 0. A first look at TeXstudio
- Added section 2.4.2 The TOC View
- Minor text changes

### CODE REVIEW

**Comment by octaeder** [comment]
Fixes #2258

File: utilities/manual/usermanual_en.html [hunk 1]
```diff
@@ -392,8 +418,25 @@ <h2 id="SECTION14">2.4 Browsing your document</h2>
 <p>
 The structure view also offers a context menu which allows one to copy/cut all text which belongs to a section (including subsection) and paste it before or after a section.
 Section can be indented/unindented which means that the hierarchy level is changed by one, i.e. \section is changed to \subsection, and all subsections are treated accordingly
+</p>
 
+<h2 id="SECTION142">2.4.2 The TOC View</h2>
+
+<p>The side panel on the left offers a TOC view (if not, s. 0. A first look at TeXstudio). The TOC shows the structure of your whole document by means of section commands (part, section, subsection, …). A mouse over shows you, in which file the section
```
Comment (sunderme) [inline utilities/manual/usermanual_en.html:425]: the reference should be a link, so the user can jump to the section easily

**Comment by sunderme** [comment]
looks good. Maybe more links, especially for the one referencing an earlier section.

**Comment by octaeder** [comment]
I had the backref \<a href="#SECTION00"\>0. A first look at TeXstudio\</a\>, was quite easy, cause a simple copy of the contents line. But since it did not work I concluded, that html jumps only forward and removed the (dead) link. Now I see that the target id was #SECTION00 instead of SECTION00.

Btw two notes: 1) My changes affect the description of bookmarks. This description is totally outdated. Funny enough it says "just click on a line number to add or remove a bookmark" (see your suggestion in #2220). 2) When I decided to add the introduction to TeXstudio I realized that all the section numbers are hard coded. Moving parts of the manual to other places or inserting new ones seems to be much work (that's why I used 0 not 1), tasks that could be done quite easily with a LaTeX document. Wouldn't a LaTeX source for the manual be a great example for the work with the superior TeXstudio?

**Pull request merged**